src/playlist-selectors.js

```diff
@@ -1,8 +1,35 @@
 import Config from './config';
 import Playlist from './playlist';
+import videojs from 'video.js';
+import window from 'global/window';
 
 // Utilities
 
+export const crossedLowWaterLine = (currentTime, buffered) => {
+  const forwardBuffer = buffered.length ?
+    buffered.end(buffered.length - 1) - currentTime : 0;
+
+  return forwardBuffer >= Config.BUFFER_LOW_WATER_LINE;
+};
+
+export const getBandwidth = (playlist, useAverageBandwidth) => {
+  if (!playlist || !playlist.attributes) {
+    return void 0;
+  }
+
+  let bandwidth;
+
+  if (useAverageBandwidth) {
+    bandwidth = playlist.attributes['AVERAGE-BANDWIDTH'];
+  }
+
+  if (!bandwidth) {
+    bandwidth = playlist.attributes.BANDWIDTH;
+  }
+
+  return bandwidth || void 0;
```

Comment:
the `|| void 0` isn't needed here since if `playlist.attributes.BANDWIDTH` doesn't exist, `bandwidth` will already be `undefined`. Though I'm ok with leaving it in for the sake of making it clear the function returns `undefined` in some cases